src/services/recipient.js

```diff
@@ -381,6 +389,10 @@ function combineObjectiveIds(existing, objective) {
   return ids;
 }
 
+export function ensureArray(input) {
+  return Array.isArray(input) ? input : [];
+}
```

Comment:
just a thought, but maybe we could move this function to constants or somewhere else

Reply:
Moved.